app/lib/katello/resources/candlepin.rb

```diff
@@ -417,7 +417,10 @@ def imports(organization_name)
 
           def pools(key, filter = {})
             if key
-              json_str = self.get(join_path(path(key), 'pools') + hash_to_query(filter), self.default_headers).body
+              params = hash_to_query(filter)
+              params += params == '?' ? '' : '&'
+              params += 'attribute=unmapped_guests_only:!true'
```

Comment:
That is subtle, and potential to break. While I tend away from comments, one here might serve well to help prevent future re-factoring breakage.